code/modules/clothing/clothing.dm

```diff
@@ -567,6 +568,21 @@
 			var/mob/living/carbon/human/H = usr
 			H.update_hair()
 
+/obj/item/clothing/head/verb/toggle_layer()
+	set name = "Switch Head Layer"
+	set category = "Object"
+	set src in usr
+
+	if(use_check_and_message(usr))
+		return 0
+	if(over_suit == -1)
```

Comment:
not a fan of this, make over_suit a boolean and then add another can_wear_over_suit variable to control whether or not it can actually have its layer changed